examples/ongoing/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources>
+    <string name="digime_application_id">IfnN9Y27Jym3P1Fad3ks3sTlo22flUBb</string>
```

Comment:
The application ID should be stripped before merging. Also, if you have a look in `ConsentActivity.kt` we do a check for this up front and bail out if it's not there with a message - you should add this here too.